apps/server/src/shared/domain/entity/submission.entity.ts

```diff
@@ -23,6 +27,10 @@ export interface ISubmissionProperties {
 @Entity({ tableName: 'submissions' })
 @Index({ properties: ['student', 'teamMembers'] })
 export class Submission extends BaseEntityWithTimestamps {
+	@ManyToOne('School', { fieldName: 'schoolId' })
+	@Index()
+	school: School;
```

Comment:
Why does the submission need to know about the school itself? I'm wondering because the task knows about the school and there is no submission without a task (isn't it?).

Reply:
I added it, becouse the key exists in the legacy model and in database collection